fsys_test.go

```diff
@@ -12,35 +15,100 @@ import (
 	"9fans.net/go/plan9/client"
 )
 
-func startAcme(t *testing.T) (*exec.Cmd, *client.Fsys) {
+func TestMain(m *testing.M) {
+	switch os.Getenv("TEST_MAIN") {
+	case "edwood":
+		main()
+	default:
+		// TODO: Replace Xvfb with a fake devdraw.
+		var x *exec.Cmd
+		switch runtime.GOOS {
+		case "linux", "freebsd", "openbsd", "netbsd", "dragonfly":
+			if os.Getenv("DISPLAY") == "" {
+				dp := fmt.Sprintf(":%d", xvfbServerNumber())
+				x = exec.Command("Xvfb", dp)
+				if err := x.Start(); err != nil {
+					log.Fatalf("failed to execute Xvfb: %v", err)
+				}
+				// Give Xvfb some time to start up.
+				// 3 seconds is default for xvfb-run.
+				time.Sleep(3 * time.Second)
```

Comment:
Do you have any experience with how flaky Travis will be with that?

Reply:
It's also 3 seconds in the [Travis CI docs](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-directly), so I doubt it'll be an issue. If it really becomes necessary, we can do a loop until `xdpyinfo` succeeds, but that would be another external dependency.